assets/styles/components/_tab-toggle.scss

```diff
@@ -159,7 +159,7 @@ $tab-active-shadow: 0 -4px 12px rgba(99, 44, 166, 0.08);
     background: #FFFFFF;
     border: 1px solid $tab-border-color;
     padding: 24px;
-    z-index: 1;
+    z-index: 0;
```

Comment:
ensures the tab content is below the tabs. removes the bottom border of the  active tab